src/interface/src/app/treatments/treatment-map/treatment-map.component.ts

```diff
@@ -152,6 +154,14 @@ export class TreatmentMapComponent {
     this.mouseStart = event;
   }
 
+  mapLoaded(event: MapLibreMap) {
+    this.mapLibreMap = event;
+  }
+
+  onImageMissing(a: any) {
+    // console.log('missing!', a);
```

Comment:
Did you want to do more with this? Or is this just a placeholder for the future?

Reply:
placeholder, kinda hard to figure this out without the final patterns